crates/libs/bindgen/src/extensions/in_addr.rs

```diff
@@ -0,0 +1,28 @@
+use super::*;
+
+pub fn gen() -> TokenStream {
+    quote! {
+        impl ::core::convert::From<::std::net::Ipv4Addr> for IN_ADDR {
+            fn from(addr: ::std::net::Ipv4Addr) -> Self {
+                Self { S_un: IN_ADDR_0 { S_addr: u32::from(addr) } }
```

Comment:
[`u32::from`](https://doc.rust-lang.org/std/primitive.u32.html#impl-From%3CIpv4Addr%3E) returns u32 in host byte order, but [`IN_ADDR`](https://docs.microsoft.com/en-us/previous-versions/windows/hardware/drivers/ff556972(v=vs.85)#remarks) awaits the address in network byte order.

Reply:
Good catch.